From 6436024137e38de7e4e8ada96fbcb2d739a46fb6 Mon Sep 17 00:00:00 2001
From: dhacker29 <dhackerdvm@gmail.com>
Date: Thu, 30 Apr 2015 04:50:33 -0400
Subject: [PATCH 2/7] Allow terminal app to show in LeanBack (1/2)

This will allow the icon to show when enabled in Developer Settings

Change-Id: I74784d20af77e4b9f143d132eca7a117caf257b8
---
 AndroidManifest.xml | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 5078f34..20627ee 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -16,6 +16,15 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
         package="com.android.terminal">
 
+    <uses-feature android:name="android.software.leanback"
+        android:required="false" />
+
+    <uses-feature android:name="android.hardware.touchscreen"
+        android:required="false" />
+
+    <uses-feature android:name="android.hardware.screen.portrait"
+        android:required="false" />
+
     <application
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_label"
@@ -29,6 +38,7 @@
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.LAUNCHER" />
+                <category android:name="android.intent.category.LEANBACK_SETTINGS" />
             </intent-filter>
         </activity>
 
-- 
2.17.1

